framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -603,14 +603,16 @@ public static String getExistingDrillStoragePlugin(String ipAddress,
    * @throws InterruptedException
    */
   public static void updateDrillStoragePlugins(String templatePath) throws InterruptedException {
-    LOG.info(">> Path: " + templatePath + "\n");
+    LOG.info("\n>> Path: " + templatePath + "\n");
     File[] templateFiles = new File(templatePath).listFiles();
     for (File templateFile : templateFiles) {
       String filename = templateFile.getName();
       LOG.info(">> Updating File: " + filename);
       String pluginType = filename.substring(0, filename.indexOf('-'));
       boolean isSuccess = Utils.updateDrillStoragePlugin(templateFile.getAbsolutePath(), pluginType, DrillTestDefaults.FS_MODE);
-      LOG.info(">> Update file " + filename + (isSuccess ? " succeeded" : " failed"));
+      if(!isSuccess){
+        LOG.info(">> Update file " + filename + (isSuccess ? " succeeded" : " failed"));
```

Comment:
if !isSuccess is true (upload fails), then there is no need for another check while printing, right?